services/src/main/java/org/fao/geonet/api/tools/i18n/TranslationApi.java

```diff
@@ -288,7 +288,7 @@ public void deleteTranslations(
                     "  \"translationKey2\": \"Translated Key Two\",\n" +
                     "  \"translationKey3\": \"Translated Key Two\"\n" +
                     "}")
-        }, schema = @Schema(type = "{ < * >: string }"))
+        }, schema = @Schema(type = "object", implementation = java.util.HashMap.class))
```

Comment:
@ianwallen, with this change I get the following in the `yml` file, is that what is expected?

```
          content:
            application/json:
              schema:
                type: string
```

Reply:
You are correct,  this does not seem correct

According to the following

https://swagger.io/docs/specification/data-models/dictionaries/

it should be 

![image](https://github.com/geonetwork/core-geonetwork/assets/1868233/696d0bf6-d129-4997-b1ff-aac01c891ac8)

I can get it to produce the following

```
          content:
            application/json:
              schema:
                type: object
                additionalProperties:
                  type: string
```
if I removed `@ApiResponse` and used `@ResponseStatus(OK)` instead. 
But in that case, the example is lost.  Soon as I add `@ApiResponse` - even if only for the example, the output is changed.

There may be bug with @ApiResponse.  I have tested with SpringDoc 1.7 and I see the same results.

So I have updated the PR to remove `@ApiResponse` and the example and replace it by `@ResponseStatus(OK)` as I believe it is better to have the correct output even if the example is losts.

I believe, we can re-investigate later on how to re-add the example.

Reply:
I have found the following related issues on Swagger-core

https://github.com/swagger-api/swagger-core/issues/3724

It seems like a possible work around would be to define a new class that extends `HashMap<String,String>`. But this does not seem ideal.

Reply:
@josegar74 
I created the following branch that shows how the changes would look if we use the workaround identified in https://github.com/swagger-api/swagger-core/issues/3724

https://github.com/ianwallen/core-geonetwork/tree/fix_spring_doc_mapping_2
https://github.com/ianwallen/core-geonetwork/pull/4

Let me know if those changes look better and I and merge those changes into this PR.

Reply:
@ianwallen https://github.com/ianwallen/core-geonetwork/tree/fix_spring_doc_mapping_2 looks good

Reply:
I already merged those changes into this PR